tests/RobotFramework/tests/cumulocity/Operation/c8y_Command_Fails

```diff
@@ -0,0 +1,5 @@
+[exec]
```

Comment:
Also be careful with the directory name. We already have tests/RobotFramework/tests/cumulocity/**operation**. You may rename from `Operation` to `custom_operation`. (just suggestion, you can use another name, but `Operation` is very much conflict.

Reply:
Yes, make sense. Renamed it.